lib/top_hangman/game.rb

```diff
@@ -9,15 +9,15 @@ class Game
 
     attr_reader :current_round, :rounds_history, :state
 
-    def initialize
-      @current_round = nil
-      @rounds_history = []
+    def initialize(current_round = nil, rounds_history = [])
+      @current_round = current_round
+      @rounds_history = rounds_history
       @state = nil
     end
 
     def start
       @state = RUNNING
-      @current_round = Round.new
+      @current_round = Round.new unless @current_round && @current_round.state == Round::RUNNING
```

Comment:
let's create a method `Round#running?` that checks internally `@state == RUNNING`, and call this method here
```suggestion
      @current_round = Round.new unless @current_round&.running?
```